test/lib/file_data_processor_test.rb

```diff
@@ -12,9 +12,48 @@ module StockMarkets
       end
     end
 
+    describe '#load_from_network' do
+      subject { file_data_processor.load_from_network }
+
+      before do
+        StockMarkets.configure do |config|
+          config.data_file_path = 'test/helpers/loaded_network_data.csv'
+        end
+      end
+
+      describe 'when source exists' do
+        before do
+          StockMarkets.configure do |config|
+            config.source_file_url = 'https://www.iso20022.org/sites/default/files/ISO10383_MIC/ISO10383_MIC.csv'
+          end
+        end
+
+        it 'returns instance of CSV::Table' do
+          assert_equal(subject.class, CSV::Table)
+        end
+      end
+
+      describe 'when source is unexisting url' do
+        before do
+          StockMarkets.configure do |config|
+            config.source_file_url = 'https://www.iso20022.org/123123123123'
+            config.data_file_path = 'unexisting'
+          end
+        end
+
+        it 'returns nil' do
+          assert_nil(subject)
+        end
+      end
+    end
+
     describe '#load_from_disk!' do
       subject { file_data_processor.load_from_disk! }
 
+      before do
```

Comment:
can we move it to the `global` describe or context and do not add in each describe section?